app/service/rpc_service.ts

```diff
@@ -35,6 +35,21 @@ class RpcService {
     (window as any)._rpcService = this;
   }
 
+  debuggingEnabled(): boolean {
+    const url = new URL(window.location.href);
+    let sp = url.searchParams.get("debug");
+    if (sp === null) {
+      let match = url.hash.match("debug=(.*)");
```

Comment:
on second though, I'm not sure this hash matching logic will work -- pages that check for `hash == <something>` will not render as expected, I think

Reply:
I can remove this bit and we can rely on the searchParams. Once we switch all routes to use /paths instead of fragments, this should be a non-issue. wdyt?